### ISSUE TITLE
Fyne app warnings

### ISSUE THREAD

**Issue by gsbhasin123** [fyne-io/fyne#629]
```
# fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw
In file included from ../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/c_glfw_linbsd.go:24:
../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/linux_joystick.c: In function ‘_glfwInitJoysticksLinux’:
../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/linux_joystick.c:224:42: warning: ‘%s’ directive output may be truncated writing up to 255 bytes into a region of size 9 [-Wformat-truncation=]
  224 |             snprintf(path, sizeof(path), "%s/%s", dirname, entry->d_name);
      |                                          ^~~~~~~
In file included from /usr/include/stdio.h:867,
                 from /usr/include/X11/Xcursor/Xcursor.h:26,
                 from ../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/x11_platform.h:39,
                 from ../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/internal.h:169,
                 from ../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/glfw/src/x11_init.c:28,
                 from ../../../go/src/fyne.io/fyne/vendor/github.com/go-gl/glfw/v3.2/glfw/c_glfw_linbsd.go:19:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:67:10: note: ‘__builtin___snprintf_chk’ output between 12 and 267 bytes into a destination of size 20
   67 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   68 |        __bos (__s), __fmt, __va_arg_pack ());
      |        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Show on my PC when running any Fyne app, but the app itself works fine. `uname -a`:
```
$ uname -a
Linux robot 5.3.0-26-generic #28-Ubuntu SMP Wed Dec 18 05:37:46 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by andydotxyz** [comment]
This compile warning is fixed upstream and will be in out 1.3 release.

Closing as duplicate of #221

**Comment by gsbhasin123** [comment]
Thanks for the quick response
